sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/state/DefaultSynchronousMetricStorage.java

```diff
@@ -131,12 +131,14 @@ public MetricData collect(
 
     // Grab aggregated points.
     List<T> points = new ArrayList<>(aggregatorHandles.size());
-    for (Map.Entry<Attributes, AggregatorHandle<T, U>> entry : aggregatorHandles.entrySet()) {
-      T point = entry.getValue().aggregateThenMaybeReset(start, epochNanos, entry.getKey(), reset);
+    for (Attributes attributes :
```

Comment:
Randomly saw this PR somehow.

Just curious if `.forEach` would work here?

https://github.com/openjdk/jdk/blob/master/src/java.base/share/classes/java/util/concurrent/ConcurrentHashMap.java#L1597

Reply:
Nice suggestion. This reads better and yields the same reduction in allocations.